OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -417,8 +417,8 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 
 						bool.TryParse(s, out server.LobbyInfo.GlobalSettings.AllowCheats);
 						server.SyncLobbyGlobalSettings();
-						server.SendMessage("{0} {1} Developer Cheats."
-							.F(client.Name, server.LobbyInfo.GlobalSettings.AllowCheats ? "allowed" : "disallowed"));
+						server.SendMessage("{0} {1} the Debug Menu."
```

Comment:
The command line offers a shortcut to access a subset of the developer menu options. "Developer Menu" covers both cases, while "Cheats" does not.